enhancements/oc/day2-add-nodes.md

```diff
@@ -0,0 +1,262 @@
+---
+title: simplify-and-unify-day2-add-node
+authors:
+  - "@andfasano"
+  - "@zaneb"
+approvers: 
+  - "@patrickdillon"
+  - "@oourfali"
+  - "@soltysh"
+reviewers: 
+  - "@rwsu"
+  - "@pawanpinjarkar"
+creation-date: 2023-11-10
+last-updated: 2023-11-10
+tracking-link: 
+  - "https://issues.redhat.com/browse/OCPSTRAT-784"
+  - "https://issues.redhat.com/browse/AGENT-682"
+see-also:
+  - "https://github.com/openshift/enhancements/blob/master/enhancements/agent-installer/agent-based-installer.md"
+replaces: []
+superseded-by: []
+---
+
+# Simplify and unify adding nodes to clusters on day 2 using agent-based installer approach
+
+## Summary
+This enhancement proposal describes how to easily add nodes to any existing non-IPI cluster using lessons and technology from the agent-based 
+installer. The user generates an ISO image via an `oc adm` command with an optional configuration file to describe the new nodes. The image 
+is used to boot the hosts so that they will be added to the cluster without any further action, except for the final approval that remains a 
+manual operation up to the user. An additional `oc adm` command will allow the user to monitor the progress of the task.
+
+## Motivation
+The task of adding a node to an already existing non-IPI cluster (regardless of the installation approach adopted) still remains a complex 
```

Comment:
If a cluster has been deployed via IPI but the user wants to use this method that should still be possible. For example you install with IPI on bare metal and then you add nodes but don't have access to their BMCs, which is a requirement for IPI bare metal deployment, this method solves this problem perfectly. So I wouldn't say this is for non-IPI clusters.

Reply:
The mentioned case is reported in the goals section, given that such specific scenario will be supported for IPI clusters, but in the other cases (especially for the public clouds) the expectation is that such approach wouldn't want to replace existing mechanisms (and also this point was added in the non-goals section)

Reply:
Right, I think the non-IPI cluster mention makes it too restrictive and makes the reader think that adding nodes to an IPI-deployed cluster is out of the scope, when it shouldn't.

Reply:
Based also on @zaneb [comment](https://github.com/openshift/enhancements/pull/1517/files#r1413310649) (below), we could rephrase it so that it would be more clear the addressed scenarios/motivations. In general I'd like to emphasize that this proposal does not intend to address/replaced already working mechanism, such as the mentioned autoscaling